salt/modules/cassandra_cql.py

```diff
@@ -361,7 +361,7 @@ def cql_query(query, contact_points=None, port=None, cql_user=None, cql_pass=Non
     return ret
 
 
-def cql_query_with_prepare(query, statement_name, statement_arguments, async=False,
+def cql_query_with_prepare(query, statement_name, statement_arguments, asynchronous=False,
```

Comment:
this will need to have `**kwargs` added, and pop `async` out of kwargs and add a warn_until message.